.travis.yml

```diff
@@ -3,9 +3,9 @@ language: node_js
 notifications:
   email: false
 node_js:
+  - '16'
   - '14'
   - '12'
-  - '10'
```

Comment:
this is a breaking change; we won't be dropping node 10.

Reply:
We're not dropping Node 10 in CI, as I've added them for all older ESLint versions

Reply:
ah ok, as manual includes. seems good.